boards/arm/nrf7002dk_nrf5340/nrf5340_cpuapp_common.dts

```diff
@@ -66,7 +66,6 @@
 			   <11 0 &gpio1 7 0>,	/* D5 */
 			   <12 0 &gpio1 8 0>,	/* D6 */
 			   <13 0 &gpio1 9 0>,	/* D7 */
-			   <14 0 &gpio1 10 0>,	/* D8 */
```

Comment:
Why?

Reply:
This is moved to the coex pins `btrf-switch-gpios`

Reply:
On nRF7002DK schematic, gpio1 pin 10 is connected to a BLE antenna switch and needs to be driven from firmware to support coexistence.

Reply:
Let me be clearer. This is not a correct change.

1. You are silently breaking the pin numbers for every pin after this in the list. There has to be an entry for D8, or D9 will silently become D8, D10 will become D9, etc.
2. It doesn't matter if the pin is also connected elsewhere on the PCB. If it is brought out to the connector at that index, that is the entry that needs to be in the phandle-array.

Reply:
got it thanks, will revert this.